### PR TITLE
71 add controller for bookings page

### PR DESCRIPTION
**Pull request by KentonMewling**
bookigns now works

### CODE REVIEW

**Review by mvallido — APPROVED** [review]
Implementation looks good

**Comment by GMcLachlan45** [comment]
This doesnt look good you need to change web.php

File: src/routes/web.php [hunk 1]
```diff
@@ -17,15 +18,6 @@
...
 
-Route::get('/', function () {
-    return Inertia::render('Welcome', [
-        'canLogin' => Route::has('login'),
-        'canRegister' => Route::has('register'),
-        'laravelVersion' => Application::VERSION,
-        'phpVersion' => PHP_VERSION,
-    ]);
-});
-
```
Comment (GMcLachlan45) [inline src/routes/web.php]: The welcome needs to be here for the app to work

Comment (GMcLachlan45) [reply src/routes/web.php]: Alrighty it looks better now thanks.

**Review by mvallido — APPROVED** [review]
Looks good now :)

**Review by GMcLachlan45 — APPROVED** [review]
Okay now you're good to go.

**Review by OneInA1E6 — APPROVED** [review]
consider them reviewed

**Pull request merged**